### ISSUE TITLE
UILayoutGuide readableContentGuide

### ISSUE THREAD

**Issue by ekazaev** [PureLayout/PureLayout#165]
Hello, guys! I have a question: will PureLayout support constraints to  UILayoutGuide readableContentGuide, which was added in iOS 9?

**Comment by mickeyreiss** [comment]
Good idea! We could definitely add support for this. Would you be willing to submit a PR?
